### PR TITLE
feat: store prompt activation date

### PR DESCRIPTION
**Pull request by leogermani**
For future analytics purposes

### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/trunk/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Stores the activation date of the prompts so we can later use for analytics purposes

### How to test the changes in this Pull Request:

1. create or activate an inactive prompt
2. Check that the `activation_date` metadata was created with the current date
3. Deactivate the prompt
4. Confirm the `dectivation_date` metadata was created with the current date
5. Activate it again
6. Confirm the activation date was updated

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: includes/class-newspack-popups.php [hunk 1]
```diff
@@ -537,6 +538,18 @@ public static function register_meta() {
 			]
 		);
 
+		\register_meta(
```
Comment (dkoo) [inline includes/class-newspack-popups.php:541]: Do we also need to register `deactivation_date`?

**Review by dkoo — APPROVED** [review]
Working as described with a non-blocking question. How do you envision using this data?

File: includes/class-newspack-popups.php (hunk 1 above)
Comment (leogermani) [reply includes/class-newspack-popups.php:541]: yes, done

**Comment by leogermani** [comment]
> Working as described with a non-blocking question. How do you envision using this data?

It is (or was) part of the Product usage measurement project scope

**Review by dkoo — APPROVED** [review]

**Pull request merged**